src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid $c-borderColor;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  color: $c-basicHeader;
```

Comment:
it's better not to use the component name for variables with color (we can use the same color for a completely different component), just set the name the same way we name the color

F.e.

```SCSS
$c-red
$c-light-blue
$c-dark-gray
```

or we can use color saturation in the name
<img width="144" alt="image" src="https://user-images.githubusercontent.com/86183007/216517467-c9b38dd8-5ad4-4ea3-8960-e58b36391ec7.png">